admin/config/locales/en.yml

```diff
@@ -1516,7 +1516,6 @@ en:
         category: Category
         category_id: Category
         category_ids: Categories
-        changes: Changes
```

Comment:
yes, I did check whether this was used elsewhere before I deleted it. thanks for asking.